tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -26,6 +31,19 @@ module('Acceptance | guid file | registration files', hooks => {
     hooks.beforeEach(function(this: ThisTestContext) {
         this.registration = server.create('registration');
         this.file = server.create('file', { target: this.registration, name: 'Test File' });
+        this.node = server.create(
+            'node',
+            {
+                _anonymized: false,
+            },
+            'currentUserAdmin',
+            'withAffiliatedInstitutions',
+            'withContributors',
+            'withDoi',
+            'withFiles',
+            'withLicense',
+            'withRegistrations',
```

Comment:
Are all these necessary for your test scenario? You should only use the ones that are relevant to the test, otherwise it will slow down tests, especially in the beforeEach function, since it will happen for each test.

Reply:
Ok, I can remove some of the the fields.